flink-cdc-common/src/main/java/org/apache/flink/cdc/common/types/utils/DataTypeUtils.java

```diff
@@ -26,10 +26,12 @@
 import org.apache.flink.cdc.common.data.ZonedTimestampData;
 import org.apache.flink.cdc.common.types.DataType;
 import org.apache.flink.cdc.common.types.DataTypes;
+import org.apache.flink.cdc.common.types.RowType;
 import org.apache.flink.cdc.common.utils.Preconditions;
 import org.apache.flink.util.CollectionUtil;
 
 import java.util.List;
+import java.util.stream.Collectors;
 
 /** Utilities for handling {@link DataType}s. */
 public class DataTypeUtils {
```

Comment:
```suggestion 
class DataTypeUtils {
```

Reply:
I get the error `Test class 'DataTypeUtilsTest' is not constructable because it is not 'public'`.

Reply:
Changing the scope will be compile failed.

Reply:
Sorry, I marked it on the wrong class.
It should be `DataTypeUtilsTest`.

Reply:
Get it, Done.